CONTRIBUTING.md

```diff
@@ -10,13 +10,31 @@ Contributions are welcome! Please see the [development section](#development) be
 
 ## Development
 
+* [Design Principles](docs/design_principles.md)
 * [Coding Guidelines](docs/coding_guidelines.md)
+* [RFC Process](#rfc-process)
 * [Development Guide](docs/developers.md)
 * [Development Roadmap](https://docs.google.com/document/d/17729Rlyo1xzlJObzHpFLDzeCVgvwRh0ktAmMEJLK-EU/edit)
 * [Fuzzing](docs/fuzzing.md)
 * [Nix](docs/nix.md)
 * [Release Process](docs/release_process.md)
 * [Tests](tests/README.md)
+* [DCO](#developers-certificate-of-origin)
+
+## RFC Process
+
+This is for "substantial" or breaking changes. Bug fixes, doc updates, small features, and issues tagged with "good first issue" can utilize the normal github pull request workflow.
+
+1. Create a new issue, where the title is prefixed with "RFC" and apply the RFC tag ([example](https://github.com/bpftrace/bpftrace/issues/2954)). Be sure to include the goal(s) of this change, potential downsides (if applicable), and other solutions you've considered. This issue is where discussions can be had about the overall design and approach. For the most part implementation details should NOT be discussed as they often lead to bike-shedding of the overall proposal.
+
+1. If the RFC is approved by a majority of the maintainers, create a POC and when you're ready, submit a pull request (linking to the original RFC). This is a good place to have others experiment with your POC and discuss implementation details. 
```

Comment:
Approval seems like a high bar for just a PoC. What about loosening the language to something like "reasonable level of interest"? I would prefer to consider a PoC as part of the RFC process - it ensures the idea is viable.

The high bar IMO should be stabilization.

Reply:
Yah good call. I'll update it to be less strict before starting a POC.